input/pages/09-b-cqlreference.md

```diff
@@ -5976,6 +5976,8 @@ define "EquivalentIsFalse": Concept1 ~ Concept2
 > Note: Because code declarations in CQL allow the value of the code to be any valid code from the code system, it is possible to define code declarations that use expressions if the code system supports it (_post-coordinated expressions_ in SNOMED, for example). However, using such features of a code system would effectively require a terminology server in order to fully support the capability, and CQL is not prescriptive about such support, and artifacts that make use of these types of features for code equivalence should come with a warning about implementability.
 {: .note-warning}
 
+{: .contains_codesystem}
+
```

Comment:
There is already an anchor here from the header, no need for this change:
https://cql.hl7.org/2025Sep/09-b-cqlreference.html#contains-codesystem